larsim/IonizationScintillation/ISCalcNESTLAr.h

```diff
@@ -30,6 +30,7 @@
 #define LAr_Z       (18)
 #define Density_LAr (1.393)
 #define LAr_W_Value (19.5*CLHEP::eV)
+#define LAr_Density (1.4)
```

Comment:
A fairly old school way of defining a value!
Could this be made as a private data member or something similar, rather than a preprocessor directive?